### PR TITLE
PROXY-4: connect express

### PR DESCRIPTION
**Pull request by i-siz**
- express framework connected
- server is starting on port, defined in the environment variables
- a message, that server is started on specified port is printed to the console
- to get the start and end dates of a period (from Monday to Friday) moment.js library is used
- environment variables are managed by dotenv library

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "This is a project for learning Node/Express. In fact it will be proxy server for original NASA API.",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "dev": "nodemon -r dotenv/config ./server.js ",
```
Comment (m-bondarev) [inline package.json]: I think you can remove nodemon and use internal node functionality 
```
"dev": "node --watch ./server.js",
```
If you want to use env variables from env file:
```
"dev": "node --watch ./server.js --env-file=.env"
```

**Review by m-bondarev — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (i-siz) [reply package.json]: Thanks. Updated.

File: index.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
 const axios = require('axios');
-const { ASTEROIDS_FEED_URL, API_KEY } = require('./environment');
+const moment = require('moment');
 
-const START_DATE = '2024-02-26';
-const END_DATE = '2024-03-01';
+const DATE_TEMPLATE = 'YYYY-MM-DD';
+
+const lastFriday = moment().day(-2);
```
Comment (mlisovich) [inline index.js:6]: Hey, this will always return last Friday base on current date? Did I understand correctly?
day(0) = Su
day(1) = Mo
day(6) = Sa

Comment (i-siz) [reply index.js:6]: Yes, you understood correctly.
Some examples you can find here:
https://www.geeksforgeeks.org/moment-js-moment-day-method/

Comment (mlisovich) [reply index.js:6]: ok cool, looks better than what I did using Date()

**Pull request merged**